frb/galaxies/frbgalaxy.py

```diff
@@ -867,6 +870,21 @@ def _make_outfile(frbname):
         outfile = '{}{}_host.json'.format(prefix, frbname)
         return outfile
 
+    def calc_dm_halo(self, mNFW=None):
+        """ Calculate the Halo contribution to the host"""
```

Comment:
define mNFW

Reply:
Replaced mnfw with a `**kwargs` argument. The `dm_host_halo` already handles the instantiation of the mNFW profile.